app-admin/ruri/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=ruri
+PKGSEC=admin
+PKGDEP="libcap libseccomp"
```

Comment:
"Zero dependency" but not really?

Reply:
https://wiki.aosc.io/zh/developer/packaging/package-styling-manual/#gou-jian-shi-yi-lai

When built as a static binary, no dependencies are required.

Reply:
> https://wiki.aosc.io/zh/developer/packaging/package-styling-manual/#gou-jian-shi-yi-lai
> 
> When built as a static binary, no dependencies are required.

Got it.